cts/cli/regression.crm_mon.exp

```diff
@@ -591,6 +591,7 @@ Operations:
 
 Negative Location Constraints:
   * not-on-cluster1	prevents dummy from running on cluster01
+No verification errors
```

Comment:
Some of the crm_mon regression tests use `--include=all`, so it's expected that they will start showing the new text. XML output defaults to `--include=all`, so that will show up in even more tests.